api/buffer/design.go

```diff
@@ -477,6 +477,7 @@ var Mapping = Type("Mapping", func() {
 
 var Column = Type("Column", func() {
 	Description("An output mapping defined by a template.")
+	Attribute("primaryKey", Boolean, "Sets this column as the primary key of the table.")
```

Comment:
Hmm, přijde mi že ten popisek tak jako implicitně vnucuje, že PK může být jen na jednom sloupci. Co třeba `Sets this column as part of the primary key..`, dávalo by to smysl? Nebo  ještě jinak?